cfgov/jinja2/v1/_includes/atoms/large_checkbox.html

```diff
@@ -0,0 +1,57 @@
+{# ==========================================================================
+
+   large_checkbox.render()
+
+   ==========================================================================
+
+   Description: Builds checkboxes fields with a large hit target.
+
+   settings (required):
+
+   settings.legend:     Name of the field.
+
+   settings.fields:     An array of all the checkbox labels.
+
+   settings.size:       The size of the field. Defaults to 1-2.
+                        Possible options: 1, 1-2, 1-4
```

Comment:
They could I think? The filters are just an expandable that contains a form that could contain anything I believe. @ajbush or @duelj would know better than me though.